test/utils/describeConformanceV5.js

```diff
@@ -71,7 +71,8 @@ function testThemeStyleOverrides(element, getOptions) {
   const render = createClientRender();
 
   describe('theme: style overrides', () => {
-    it("respect theme's styleOverrides custom state", () => {
+    it("respect theme's styleOverrides custom state", function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) this.skip();
```

Comment:
Can you put these in braces so that we don't miss this.

Reply:
Done